src/main/java/net/sf/jabref/gui/importer/fetcher/EntryFetcher.java

```diff
@@ -8,6 +8,7 @@
 import net.sf.jabref.logic.importer.OutputPrinter;
 
 /**
+ * <b>use SearchBasedFetcher instead</b> <br>
```

Comment:
The comment as  javadoc is better, because when you see a method call only marked as deprecated you usually don't know what to use as replacement. And when you do openDecleration and hover over the type, the javadoc shows directly the alternative